JavaScript/JavaScriptSDK/Logging.ts

```diff
@@ -43,12 +55,7 @@
                 throw message;
             } else {
                 _InternalLogging.warn(message);
-
-                if (_InternalLogging.verboseLogging() || severity === LoggingSeverity.CRITICAL) {
-                    if (this.queue.length < this.MAX_QUEUE_SIZE) {
-                        this.queue.push(_InternalLogging.AiNonUserActionable + message);
-                    }
-                }
+                this._throttle(severity, this.AiNonUserActionablePrefix + message, this.MAX_ALLOWED_EVENT_LIMIT);
             }
```

Comment:
As someone reading this code for the first time, I am confused and have to look up the actual implementation of the `warn` and `_throttle` methods to understand what's going on here. My intuitive understanding of `warn` is that it writes a message to the queue with `LoggingSeverity.WARNING`; what it actually does is write the given message to the `console`, when it's available. Intuitively, the term `_throttle` implies that the function does something to the _existing_ messages in the `queue`, whereas it adds a _new_ message to the queue, if it fits severity and throttling limits. Can we make the method names more descriptive?